ui/ui-components/client/client.ts

```diff
@@ -105,7 +105,8 @@ export class Client {
     useCache = true,
     uploadHandler?,
     req?,
-    res?
+    res?,
+    timeout?: number
```

Comment:
@seancdavis I don't think we ended up needing the timeout maybe?  that might have been a red herring before we realized that the api server wasn't rebooting properly.

Reply:
Worth a test, anyways.